# array of beacon observation
Header header
float32 angle_min
float32 angle_max
float32 angle_increment
float32 range_min
float32 range_max
Fiducial[] observations
